consensus/ibft/signer/extra.go

```diff
@@ -62,6 +92,14 @@ func (i *IstanbulExtra) MarshalRLPWith(ar *fastrlp.Arena) *fastrlp.Value {
 	// ParentCommittedSeal
 	if i.ParentCommittedSeals != nil {
 		vv.Set(i.ParentCommittedSeals.MarshalRLPWith(ar))
+	} else if i.RoundNumber != nil {
+		vv.Set(ar.NewNull())
+	}
```

Comment:
Isn't it enough to remove this part (if condition below is anyways going to set round number if it is not nil)?

This way it seems like `ParentCommittedSeals` are in some relation with `RoundNumber`, but I don't see how they relate.

```suggestion
	}
```

Reply:
Fixed 0fd2657058e238256b737ad5dcafc250f0f4c813